app/components/DataContext/DataContext.tsx

```
@@ -163,34 +161,34 @@ const MineralDataContext: React.FC<mineralContextProps> = (props) => {
                               {item.ids.map((id) => {
-                                let _isHovered = selectedIds.length && !selectedIds.includes(id);
+                                let _isHovered = !!activeIds.length && !activeIds.includes(id);
                                 let _color = minerals.find(mineral => mineral.id === id)?.color;
 
                                 return (
                                   <Dot key={id} color={_color} isHovered={_isHovered} />
                                 )}
                               )}
                             </div>
-                            <span className={cx(styles.propItem, hoverClass, _isHovered ? 'opacity-20' : '')} dangerouslySetInnerHTML={{ __html: item.value }}></span>
+                            <span className={cx(styles.propItem, hoverClass, _isHovered && 'opacity-20')} dangerouslySetInnerHTML={{ __html: item.value }}></span>
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)